src/QsCompiler/SyntaxProcessor/CapabilityInference/Capabilities.fs

```diff
@@ -18,57 +16,11 @@ let syntaxAnalyzer callableKind =
                       ResultAnalyzer.analyzer callableKind
                       TypeAnalyzer.analyzer ]
 
-// TODO: Remove this function as part of https://github.com/microsoft/qsharp-compiler/issues/1448.
```

Comment:
Could you also update Pattern to remove the 'Props type parameter and the Properties field? It shouldn't be needed anymore:

https://github.com/microsoft/qsharp-compiler/blob/f5f492db61baed935e9d1c2bcb5662f06856c6b0/src/QsCompiler/SyntaxProcessor/CapabilityInference/Analyzer.fsi#L23-L29